torch/_torch_docs.py

```diff
@@ -7327,6 +7327,53 @@ def merge_dicts(*dicts):
 """.format(**factory_common_args))
 
 
+add_docstr(torch.vandermonde_matrix,
+           """
+vandermonde_matrix(x, n, increasing=True, dtype=None, \
+layout=torch.strided, device=None, requires_grad=False) -> Tensor
+""" + r"""
+Generate a Vandermonde matrix.
```

Comment:
Generate -> Generates. Comments/docs should be written as if describing the code/function.